e2e/__snapshots__/extendConfig.test.js.snap

```diff
@@ -1341,11 +1337,6 @@ Object {
         "port": 7050,
       },
     ],
-    "organization": Object {
-      "domain": "root.com",
-      "mspName": "OrdererMSP",
-      "name": "Orderer",
-    },
```

Comment:
hm... don't get it - why it's removed from extendConfig ?
Shouldn't extendConfig show configuration after all defaults applied?